Classes/Console/Command/Database/DatabaseExportCommand.php

```diff
@@ -64,7 +64,7 @@ protected function createNativeDefinition(): array
                 'exclude',
```

Comment:
Wouldn't it make sense to have this related to `database:import` and vice-versa?

Reply:
might make sense yes. this PR is mainly to recover the functionality, not to go through all commands and see which relations could be added or removed.